php/base/newrelic.ini

```diff
@@ -0,0 +1,858 @@
+;          please note that even if you are not letting the agent start the
+;          daemon and are still using newrelic.cfg (see below) the license
+;          keyword in that file is no longer obeyed. Instead the agent will
+;          use the preserved value of that license from the key file.
+;          Once you have updated your INI files to contain the license we
+;          urge you to remove /etc/newrelic/upgrade_please.key in order to
+;          eliminate the potential for confusion about exactly where the key
+;          is coming from.
+;
+newrelic.license = "${NEW_RELIC_LICENSE_KEY}"
+
+; Setting: newrelic.logfile
+; Type   : string
+; Scope  : system
+; Default: none
+; Info   : Sets the name of the file to send log messages to.
+;
+newrelic.logfile = "/var/log/newrelic/php_agent.log"
+
+; Setting: newrelic.loglevel
+; Type   : string
+; Scope  : system
+; Default: "info"
+; Info   : Sets the level of detail to include in the log file. You should
+;          rarely need to change this from the default, and usually only under
+;          the guidance of technical support.
+;          Must be one of the following values:
+;            always, error, warning, info, verbose, debug, verbosedebug
+;
+;newrelic.loglevel = "info"
+
+; Setting: newrelic.high_security
+; Type   : boolean
+; Scope  : system
+; Default: false
+; Info   : Enables high security for all applications. When high security is
+;          enabled, the following behavior will take effect:
+;          * Raw SQL strings will never be gathered, regardless of the value of
+;            newrelic.transaction_tracer.record_sql.
+;          * Request parameters will never be captured, regardless of the
+;            newrelic.attributes configuration settings.
+;          * The following API functions will have no effect, and will return
+;            false:
+;            newrelic_add_custom_parameter
+;            newrelic_set_user_attributes
+;            newrelic_record_custom_event
+;
+;          IMPORTANT:  If you change this setting, you must also change the RPM
+;          UI security setting. If the two settings do not match, then no data
+;          will be collected.
+;
+;          IMPORTANT: This setting is not compatible with 
+;          newrelic.security_policies_token. Only one may be set. If both are
+;          set an error will be thrown and the agent will not connect. 
+;
+;newrelic.high_security = false
+
+; Setting: newrelic.appname
+; Type   : string
+; Scope  : per-directory
+; Default: "PHP Application"
+; Info   : Sets the name of the application that metrics will be reported into.
+;          This can in fact be a list of up to 3 application names, each of
+;          which must be separated by a semi-colon. The first name in any such
+;          list is considered the 'primary' application name and must be unique
+;          for each account / license key.
+;
+newrelic.appname = "${NEW_RELIC_APP_NAME}"
```

Comment:
Is this environment variable evaluated at runtime?

Reply:
We should document this envvar somewhere. Maybe skpr/example ?

Reply:
https://github.com/skpr/example/commit/b413db157ed41ffafe55c976918a2eb0269af387

Reply:
I have also pushed these as ENV items on the PHP Dockerfile